mcf-browser/src/Home.tsx

```diff
@@ -78,9 +83,9 @@ interface HomeStateType{
   */
 class Home extends Component<HomePropType, HomeStateType> {
   /** Constructor for class, sets initial state
-   *
-   * @param {Object} props the props passed in by parent component
-   */
+    *
```

Comment:
these indentations also should not have been changed?

Reply:
Fixed!